### ISSUE TITLE
Add support  for radio coprocessor and IPCC on stm32wb55

### ISSUE THREAD

**Issue by OueslatiGhaith** [embassy-rs/embassy#1397]
Hello, 

I noticed that embassy does not support IPCC or the radio coprocessor on stm32wb55. Is there any plans to add them in the future?

This would be a useful feature for many users who want to use Bluetooth or Zigbee on their stm32wb55 devices.

I have noticed that this issue was addressed by  [GH-24 ](https://github.com/embassy-rs/embassy/pull/24) though it was closed. The example there does work with RTIC although it is quite outdated

**Comment by Dirbaio** [comment]
Yes, this would be a great feature to have. No one is working on it or plans to AFAIK though. So PR's welcome :) 

The code in #24 is probably a good starting point, yes!
